processor/k8sprocessor/kube/kube.go

```diff
@@ -102,13 +102,15 @@ type FieldFilter struct {
 // ExtractionRules is used to specify the information that needs to be extracted
 // from pods and added to the spans as tags.
 type ExtractionRules struct {
-	Deployment bool
-	Namespace  bool
-	PodName    bool
-	PodUID     bool
-	Node       bool
-	Cluster    bool
-	StartTime  bool
+	Deployment    bool
+	Namespace     bool
+	PodName       bool
+	PodUID        bool
+	Node          bool
+	Cluster       bool
+	StartTime     bool
+	HostName      bool
+	ContainerName bool
```

Comment:
Would `ContainerNames` make more sense as we add all names we see?

Reply:
I've changed this but taking into account @dmitryax's comment I'm not sure it will fit since the tag `k8s.container.name` is in singular.

Reply:
I think we can keep it singular here for consistency, even if it may have multiple values

Reply:
👍🏻  done. Singular it is.